lib/sql.ts

```diff
@@ -5,6 +5,8 @@
 import {QueryProcessManager} from './process-manager';
 import type * as sqlite from 'better-sqlite3';
 import {FS} from './fs';
+// @ts-ignore in case not installed
```

Comment:
In which situation would this not be installed?

Reply:
If you don't want to use it? It's optional.

Reply:
It's optional? It's a library... Devs should probably have everything installed.

Reply:
I thought we wanted SQLite stuff to be optional?